src/ast/attachpoint_parser.cpp

```diff
@@ -778,5 +781,21 @@ AttachPointParser::State AttachPointParser::iter_parser()
   return OK;
 }
 
+AttachPointParser::State AttachPointParser::raw_tracepoint_parser()
+{
+  if (parts_.size() != 2)
+  {
+    if (ap_->ignore_invalid)
+      return SKIP;
+
+    errs_ << ap_->provider << " probe type requires 1 argument" << std::endl;
+    return INVALID;
+  }
+
+  ap_->func = parts_[1];
```

Comment:
We should explicitly disable wildcards for now:
```suggestion
  ap_->func = parts_[1];
  
  if (has_wildcard(ap_->func))
  {
    if (ap_->ignore_invalid)
      return SKIP;

    errs_ << ap_->provider << " probe type does not support wildcards"
          << std::endl;
    return INVALID;
  }
```